### PR TITLE
[F] Send userId and userName on data control task request submission

### PR DESCRIPTION
**Pull request by JLucasM**
LRAC-14388: [Jira Issue](https://liferay.atlassian.net/browse/LRAC-14388)

### CODE REVIEW

**Comment by JLucasM** [comment]
ci:test:sf

**Comment by JLucasM** [comment]
ci:test:relevant

**Comment by JLucasM** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx [hunk 1]
```diff
@@ -263,7 +263,9 @@ const RequestList: React.FC<IRequestListProps> = ({
 						emailAddresses,
 						fileName,
 						ownerId: currentUser.id,
-						types
+						types,
+						userId: currentUser.userId,
+						userName: currentUser.screenName
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx:268]: Hey @lesliewong92 can we use screenName from the logged user here?

**Comment by interaminense** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx (hunk 1 above)
Comment (interaminense) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx:268]: Get username from `currentUser.name` instead.

Comment (JLucasM) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx]: Thanks for you review @interaminense, done!

**Comment by interaminense** [comment]
ci:forward

**Comment by marcellustavares** [comment]
@interaminense @JLucasM the user name should hold the same value as https://github.com/liferay-ac/liferay-portal/blob/6729ceb13ee0a66c5b5f53350767227454280085/modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/queries/CustomAssetsReportMutation.js#L13

**Comment by interaminense** [comment]
> @interaminense @JLucasM the user name should hold the same value as
> 
> https://github.com/liferay-ac/liferay-portal/blob/6729ceb13ee0a66c5b5f53350767227454280085/modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/queries/CustomAssetsReportMutation.js#L13

Hey @marcellustavares, we're using the same value from CustomAsset, it is just renamed to `modifiedByUsername`.

Actually `modifiedByUsername` is getting from `currentUser.name`

You can take a look at it here https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/assets/custom-asset/pages/Dashboard.jsx#L126

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
Hey @JLucasM, the failure of `relevant` is not related to your modifications. The PR looks good to be forwarded. Thanks!

**Comment by JLucasM** [comment]
ci:forward:force

**Pull request closed without merging**